### DISCUSSION TITLE
Retrieving Wallet TX history Issues

### DISCUSSION THREAD

**Discussion by Adamb83** [romanz/electrs#1111] [Q&A] [answered]
Newbie here
Im using electrs, im all synced up and running with my node. Running 0.10.7 and trying to query wallet activity with no luck. 

I've tried
adam@HP:~$ echo '{"jsonrpc": "2.0", "method": "blockchain.scripthash.get_history", "params": ["c6a3233e8a97e3ea58bf520d293186ea69278dd4"], "id": 1}' | nc 192.168.30.1 50001
{"error":{"code":-32602,"message":"invalid params"},"id":1,"jsonrpc":"2.0"}

Can anyone let me know how to query the wallet histories. My electrs db is 60gb so pretty sure its all synced.. 

Many thanks for any help

**Comment by antonilol** [comment, marked as answer]
You should put in a script hash (sha256 of the output script), your string is only 20 bytes (40 hex characters) long, sha256 output is 32 bytes. There are examples in this repo in the [contrib](https://github.com/romanz/electrs/tree/master/contrib) directory, take a look at history.py.
